packages/rdbg/rdbg.1.0/opam

```diff
@@ -0,0 +1,24 @@
+opam-version: "1.2"
+maintainer: "erwan.jahier@imag.fr"
+bug-reports: "erwan.jahier@imag.fr"
+authors: [ "Erwan Jahier" ]
+license: "GPL-3"
+homepage: "http://rdbg.forge.imag.fr/"
+dev-repo: "https://forge.imag.fr/anonscm/git/rdbg/rdbg.git"
+build: [
+  ["oasis" "setup"]
+  ["ocaml" "setup.ml" "-configure" "--prefix" prefix]
+  ["ocaml" "setup.ml" "-build"]
+]
+install: ["ocaml" "setup.ml" "-install"]
+remove: [
+  ["ocamlfind" "remove" "rdbg-plugin"]
+]
+build-doc: [ "ocaml" "setup.ml" "-doc" ]
+depends: [
+  "base-unix"
+  ("camlp4" {= "4.03.0"} | "camlp4" {>= "4.02.1+system"} | "camlp4" {= "4.02.1+2"} | "camlp4" {= "4.02.1+1"} | "camlp4" {= "4.02.1"} | "camlp4" {= "4.02.0-1modular-implicits"} | "camlp4" {= "4.02.0+system"} | "camlp4" {= "4.02.0+2"} | "camlp4" {= "4.02.0+1"} | "camlp4" {= "4.02.0"} | "camlp4" {= "4.01.0"})
```

Comment:
Thanks for the clarifications David.

ps : for some reasons, I had to reinstall oasis2ocaml, and now it 
generates a good opam file in the first place. So my guess is that I've 
used a too old version.

pps : as far as the mac build failure is concerned, I'm not waiting for 
any prompting but to find a mac os machine where I could test it 
locally. Gnuplotrif is a module that is part of the package, so I guess 
it should work on mac os too. I'll re-submit a pull request when i'm 
done.

-- 
R1.